### PR TITLE
builder for AddressBlock

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
cc @Emilgardis

File: svd-rs/src/addressblock.rs [hunk 1]
```diff
@@ -60,3 +61,105 @@ impl AddressBlockUsage {
...
+            Ok(())
+        }
+    }
+    /// Validate the [`AddressBlock`].
+    ///
+    /// # Notes
+    ///
+    /// This doesn't do anything.
+    pub fn validate(&mut self, _lvl: ValidateLevel) -> Result<(), SvdError> {
+        // TODO
```
Comment (Emilgardis) [inline svd-rs/src/addressblock.rs:162]: How is this a todo? What is there to do ?

Comment (burrbull) [reply svd-rs/src/addressblock.rs:162]: Don't know yet. Delete?

Comment (Emilgardis) [reply svd-rs/src/addressblock.rs:162]: Yeah I don't think there's a reason for this note 😅

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**